pulp_smash/tests/pulp3/file/api_v3/test_sync.py

```diff
@@ -85,3 +86,70 @@ def do_test(self, download_policy, sync_mode):
         sync_repo(self.cfg, importer)
         repo = client.get(repo['_href'])
         self.assertNotEqual(latest_version_href, repo['_latest_version_href'])
+
+
+class SyncChangeRepoVersionTestCase(unittest.TestCase):
+    """Verify whether sync of repository updates repository version."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables."""
+        cls.cfg = config.get_config()
+
+    def test_all(self):
+        """Call :meth:`do_test` with varying arguments.
```

Comment:
Is there any special reason to call `do_test` with each possible pairing of `download_policy` and `sync_mode`? Doing this will make the test case consume 2-6 times as much time. One could execute the test with any valid combination of `download_policy` and `sync_mode`, and still prove that repository versions correctly increment.